### ISSUE TITLE
[pkgs] abort jobs execution with error if invalid operators are used

### ISSUE THREAD

**Issue by j-h-scheufen** [monax/monax#1135]
Problem:
An `assert` job in EPM of the following example uses an invalid operator `neq` (correct: `ne`). The job logs as `Executing`, but there is no further output of the assertion and it proceeds with the next job.

Expectation: 
Using an invalid comparison operator should cause the job execution to fail with an error.

Jobs:
```
- name: note1EndorsedBefore
  job:
    query-contract:
      destination: $note1
      abi: PromissoryNote
      data: isEndorsed

- name: assertEndorsement1Before
  job:
    assert:
      key: $note1EndorsedBefore
      relation: neq
      val: "true"
```
Output:
```
Executing Job                                 note1EndorsedBefore
Return Value                                  true
Executing Job                                 assertEndorsement1Before
Executing Job                                 runEndorsement1
```

**Comment by VoR0220** [comment]
? Shouldnt it be "ne"?

**Comment by j-h-scheufen** [comment]
Ah, that might be true. @VoR0220. It could mean the tests pass, because the assertion was never executed ... quite a trap. If I can confirm that it was a "user" mistake, how much effort would it be to make sure EPM spots invalid instructions and produces an error?

**Comment by VoR0220** [comment]
Already taken care of in my refactor ☺

**Comment by j-h-scheufen** [comment]
@VoR0220 You were right. This error had not been detected in our scripts, because it looked as if the epm.yaml tests were succeeding. I amended the ticket description to request better handling of invalid operators.

**Comment by zramsay** [comment]
@VoR0220 is this in #1323 ?

**Comment by VoR0220** [comment]
Yes, I got around it by creating the JobRunner's new `TypeType` and redid asserts so that they go through type changes, comparisons and whatnot, and made it generally flexible at to what is input for an assert.

**Comment by zramsay** [comment]
closed in #1419
